policy/modules/system/locallogin.te

```diff
@@ -154,6 +154,7 @@ ifdef(`init_systemd',`
 ')
 
 ifdef(`distro_debian',`
+	auth_read_pam_motd_files(local_login_t)
```

Comment:
It seems that it would make more sense to add this to `auth_login_pgm_domain()` so any login can read these, no?